src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventsControllerCourtLogsTest.java

```diff
@@ -176,6 +176,14 @@ void courtLogsGetResultMatch() throws Exception {
         var event = createEventWith(LOG, "test", hearingEntity, createOffsetDateTime("2023-07-01T10:00:00"));
         dartsDatabase.save(event);
 
+        // generate a hearing that we do not expect to be returned
```

Comment:
Is it worth adding abit more detail around what we are trying to achieve here? Or just link it back to the spike Jira ticket?

Reply:
Fixed